src/lightning/pytorch/strategies/xla.py

```diff
@@ -95,11 +93,6 @@ def root_device(self) -> torch.device:
 
         return xm.xla_device()
 
-    def connect(self, model: "pl.LightningModule") -> None:
-        # this is called in the spawned process, so no need to use `xmp.MpModelWrapper`
```

Comment:
The MpModelWrapper got removed a while ago, so our step redirection through _LightningModuleWrapperBase was anyway pointless.